food.go

```diff
@@ -86,6 +86,11 @@ func (f *Food) Install() error {
 	if err := os.MkdirAll(barrelDir, 0755); err != nil {
 		return err
 	}
+	// Add a conf file with the rig that the file comes from.
+	err = ioutil.WriteFile(strings.Join([]string{Home(HomePath).Barrel(), f.Name, "rig.conf"}, "/"), []byte(f.Rig), 0644)
```

Comment:
There's a few issues in this line of code:

- you could've used the `if err := ...; err != nil {` convention to collapse lines 90 and 91 into a single line.
- `"rig.conf"` should've been made a constant value which other packages could consume. That way, if `"rig.conf"` ever gets changed, I wouldn't have to go and make that same change in `cmd/gofish`. [Keep it DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).
- Using a forward slash paired with `strings.Join` means it'll only work on Linux. In the future, consider using `filepath.Join` instead.